controller/src/beerocks/master/db/db.h

```diff
@@ -136,7 +136,8 @@ class db {
         bool client_optimal_path_roaming_prefer_signal_strength = false;
     } sDbMasterSettings;
 
-    db(sDbMasterConfig &config_, beerocks::logging &logger_) : config(config_), logger(logger_)
+    db(sDbMasterConfig &config_, beerocks::logging &logger_, const std::string &local_bridge_mac)
+        : config(config_), logger(logger_), m_local_bridge_mac(local_bridge_mac)
```

Comment:
Although this address is indeed the local bridge address at the moment, we will have to change this at some point (because it's wrong: it should be a locally administered address according to 1905.1, while the bridge address is usually a OUI address). Therefore, I wouldn't create a new variable that has "bridge" in the name. Instead, use `al_mac`, `m_al_mac` etc.

Reply:
+1 for `m_al_mac`

Reply:
I don't understand, in the future `al_mac` and the bridge mac won't be equal?

Reply:
The name should be al_mac. Each device can define its own al mac, currently it is the bridge mac but it can change, so the answer is "maybe".